featuregates/Makefile

```diff
@@ -19,10 +19,22 @@ CRD_OPTIONS ?= "crd"
 
 all: manager
 
+ENVTEST_K8S_VERSION = 1.26.1
+ENVTEST ?= $(LOCALBIN)/setup-envtest
+
+LOCALBIN ?= $(shell pwd)/bin
```

Comment:
binaries should be downloaded to `hack/tools/bin` in project root directory, that's where we have been downloading the binaries needed for tanzu-framework

Reply:
Done